prac_05/hex_colours.py

```diff
@@ -0,0 +1,17 @@
+"""
+CP1404/CP5632 Practical 05
+Look up hexadecimal codes
+"""
+
+NAME_TO_CODE = {"absolutezero": "0048ba", "acidgreen": "#b0bf1a", "aliceblue": "#f0f8ff", "amaranth": "#e52b50",
```

Comment:
the same comment as before, I don't find this naming helpful, partially if you consider it was a part of a larger program.

Reply:
Again, I was just following the style guide.